charts/athens-proxy/templates/deployment.yaml

```diff
@@ -54,20 +54,10 @@ spec:
           periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
           successThreshold: {{ .Values.livenessProbe.successThreshold }}
           timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
-            {{- if .Values.basicAuth.enabled }}
-            httpHeaders:
-              - name: Authorization
-                value: "Basic {{ (printf "%s:%s" (default "gomod" .Values.basicAuth.user) (default "gomod" .Values.basicAuth.password) | b64enc) }}"
-            {{- end }}
         readinessProbe:
           httpGet:
-            path: "{{ template "readinessPath" . }}"
+            path: "/readyz"
```

Comment:
@bmcstdio would you be ok with leaving this as-is with the `{{template ...}}`?

Reply:
@arschles sure thing, I've reverted that.